### ISSUE TITLE
GLB model issue

### ISSUE THREAD

**Issue by LeXXik** [playcanvas/engine#5420]
Forwarding from a Discord user.

There is an issue with a GLB model. It looks like the source model can be viewed in the model viewer correctly, but the parsed container doesn't look right.

https://playcanvas.com/project/1093092/overview/model-issue

**Comment by yaustar** [comment]
I get the following error on the browser console:

```
1780843:1 [.WebGL-0x11808f59d00] GL_INVALID_OPERATION: Vertex buffer is not big enough for the draw call
```

**Comment by mvaligursky** [comment]
Agreed, the EXT_meshopt_compression is required by this glb, which is not supported to import to the Editor. You can add support for it for dynamic loading from a url, similarly to how the viewer does though.

Closing as this is not an issue. We have a tracking issue for this here already: https://github.com/playcanvas/engine/issues/2630
